Makefile

```diff
@@ -151,26 +152,27 @@ crossbuild:
 	@echo "$$CROSSBUILD_HELP_INFO"
 else
 crossbuild: clean
-	hack/make-rules/crossbuild.sh $(WHAT) $(GOARM)
+	ARCH=$(ARCH) hack/make-rules/crossbuild.sh $(WHAT)
 endif
 
-
-
 SMALLBUILD_COMPONENTS=edgecore \
 	edgesite
 define SMALLBUILD_HELP_INFO
-# small build components.
+# Small build components.
 #
 # Args:
-#   WHAT: Component names to be lint check. support: $(SMALLBUILD_COMPONENTS)
-#         If not specified, "everything" will be small build.
+#   WHAT: binary name to build. support: $(BINARIES)
```

Comment:
should be careful here, need to make sure smallbuild works with all those binaries, this used to be "edgecore" and  "edgesite" only.

Reply:
In fact smallbuild is a pack process, it works for all binaries.